### DISCUSSION TITLE
laravel log file information

### DISCUSSION THREAD

**Discussion by elsoftpy** [DirectoryTree/LdapRecord#500] [Q&A] [answered]
Hello:
I found this line in my laravel.log file 

> [2022-09-12 16:18:28] local.INFO: LDAP (ldap://xxxxxxx) - Operation: Attempting - Username: xxxx

 Does this log come from the package? and if it's from the package, how can I configure it to not show it in production?

Thank you

**Comment by stevebauman** [comment, marked as answer]
Hi @elsoftpy,

Yes the log entry you've shown is from LdapRecord. To turn it off in production, set the `LDAP_LOGGING` env variable to `false`:

```env
LDAP_LOGGING=false
```

https://ldaprecord.com/docs/laravel/v2/configuration#using-an-environment-file-env

**Reply by elsoftpy** [reply]
Thank you very much.

**Reply by stevebauman** [reply]
Happy to help @elsoftpy ❤️
